### PR TITLE
Added method for setting and inserting the offering code into database

### PR DESCRIPTION
**Pull request by sirianishetty**

### CODE REVIEW

File: DCode.Services/Common/CommonService.cs [hunk 1]
```diff
@@ -581,5 +583,16 @@ public List<string> GetFINotificationRecipientsForServiceLine(int serviceLineId)
                 ?.Split(',')
                 ?.ToList();
         }
+
+        public void SetOfferingCodeAndInsertIntoDb(IEnumerable<offering> offerings)
+        {
```
Comment (shivam-r) [inline DCode.Services/Common/CommonService.cs:588]: Can you please optimise this code. Don't use ToList() with each reference to offerings. Please set it in local variable once before going into loop and use that temp variable.

**Review by ritwiksen — CHANGES_REQUESTED** [review]
As discussed offerings and portfolio tables are readonly. Please make the modifications as necessary.

File: DCode.Services/Common/CommonService.cs [hunk 2]
```diff
@@ -581,5 +583,16 @@ public List<string> GetFINotificationRecipientsForServiceLine(int serviceLineId)
                 ?.Split(',')
                 ?.ToList();
         }
+
+        public void SetOfferingCodeAndInsertIntoDb(IEnumerable<offering> offerings)
```
Comment (shivam-r) [inline DCode.Services/Common/CommonService.cs:587]: Why do we need this method? Offerings will already have a predefined code.

**Review by shivam-r — CHANGES_REQUESTED** [review]

**Comment by ritwiksen** [comment]
Closing this PR as the functionality is already in place.

**Pull request closed without merging**